ch_tools/monrun_checks/ch_orphaned_objects.py

```diff
@@ -0,0 +1,90 @@
+from datetime import timedelta
+from typing import Optional
+
+import click
+
+from ch_tools.common.cli.parameters import TimeSpanParamType
+from ch_tools.common.clickhouse.config import get_clickhouse_config
+from ch_tools.common.commands.clean_object_storage import DEFAULT_GUARD_INTERVAL, clean
+from ch_tools.common.result import CRIT, OK, WARNING, Result
+
+DRY_RUN = True
+ON_CLUSTER = True
+CLUSTER_NAME = "{cluster}"
+DISK_NAME = "object_storage"
+OBJECT_NAME_PREFIX = ""
+
+
+@click.command("orphaned-objects")
+@click.option(
+    "--keep-paths",
+    "keep_paths",
+    is_flag=True,
+    help=(
+        "Do not delete the collected paths from the local table, when the command completes."
+    ),
+)
+@click.option(
+    "--use-saved-list",
+    "use_saved_list",
+    is_flag=True,
+    help=("Use saved object list without traversing object storage again."),
+)
+@click.option(
+    "-c", "--critical", "crit", type=int, default=3600, help="Critical threshold."
+)
+@click.option(
+    "-w", "--warning", "warn", type=int, default=600, help="Warning threshold."
```

Comment:
Let's set more reasonable default values, for example, warn: `100 * 1024**2`, crit: `10 * 1024**3`